__tests__/Car.test.js

```diff
@@ -0,0 +1,30 @@
+import Car from "../src/domain/Car.js";
+import * as Util from "../src/Util.js";
+
+jest.mock("../src/Util.js");
+
+test("랜덤값이 4이상이면 앞으로 한칸 전진한다", () => {
+  //givin
+  Util.getRandomNumber
+    .mockReturnValueOnce(4)
+    .mockReturnValueOnce(2)
+    .mockReturnValueOnce(9);
+
+  //when
+  const car = new Car();
+  car.tryMove();
+  expect(car.position).toBe(1);
+
+  car.tryMove();
+  expect(car.position).toBe(1);
+
+  car.tryMove();
+  expect(car.position).toBe(2);
+});
+
+test("자동차에 이름을 부여할 수 있다", () => {
+  //given,when
+  const car = new Car("캉골");
+  //then
+  expect(car.raceCarName).toEqual("캉골");
+});
```

Comment:
어떤걸 테스트 하고 있는지 Context(맥락)을 알 수 있게 해주면 좋을거 같아요.

```suggestion
describe("Car 클래스 테스트", () => {
  test("랜덤값이 4이상이면 앞으로 한칸 전진한다", () => {
    //givin
    Util.getRandomNumber
      .mockReturnValueOnce(4)
      .mockReturnValueOnce(2)
      .mockReturnValueOnce(9);

    //when
    const car = new Car();
    car.tryMove();
    expect(car.position).toBe(1);

    car.tryMove();
    expect(car.position).toBe(1);

    car.tryMove();
    expect(car.position).toBe(2);
  });

  test("자동차에 이름을 부여할 수 있다", () => {
    //given,when
    const car = new Car("캉골");
    //then
    expect(car.raceCarName).toEqual("캉골");
  });
});

```

Reply:
https://github.com/woowacourse/javascript-racingcar/pull/365/commits/4b75e12ae1d9919786c99d2e105cd9c553a1c602
car 클래스 test 코드를 전체적으로 수정하였습니다